includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -139,8 +160,8 @@ public function javascript_params() {
 		global $wp;
 
 		$stripe_params = [
-			'publishableKey' => $this->publishable_key,
 			'isUPEEnabled'   => true,
+			'key'            => $this->publishable_key,
 			'locale'         => WC_Stripe_Helper::convert_wc_locale_to_stripe_locale( get_locale() ),
```

Comment:
Is there a reason we changed this here instead of updating the JS?

To be clear; I'm fine with this change, I just wonder if it wouldn't be better to re-use the parent JS function and use the same configuration object throughout the whole JS codebase instead of having different names for each configuration object depending on whether you're working in the Blocks, Shortcode, or UPE JS 🤔

Reply:
Both the JS and the PHP were changed to use `key` instead of `publishableKey`.
The reason is that the current Stripe gateway uses `key` for both shortcode and blocks checkouts... the name `publishableKey ` was copied from WC Payments upe javascript.

Reply:
Ah, sorry about that. I didn't realize this was aligning the copied WCPay code with pre-existing Stripe configuration. Ignore this comment then 🙂